optimizely/optimizely_config.py

```diff
@@ -133,7 +133,7 @@ def __init__(self, project_config):
             optly_audience = OptimizelyAudience(
```

Comment:
If you want, you can take a ref of typed_audiences using `[]`.

Reply:
Actually going to make it simpler and just use project_config.typed_audiences reference in the loop rather than assign something new. Was doing things a bit differently previously and never cleaned up. Thanks for spotting.